docs/visualization/normalization.rst

```diff
@@ -260,3 +260,44 @@ to be used in scripted programs; it's better to use
     ax = fig.add_subplot(1, 1, 1)
     im = ax.imshow(image, origin='lower', norm=norm)
     fig.colorbar(im)
+
+Combining stretches and Matplotlib normalization
+================================================
+
+Stretches can also be combined with other stretches, just like transformations.
+The resulting :class:`~astropy.visualization.stretch.CompositeStretch` can be 
+used to normalize Matplotlib images like any other stretch. For example, a
+composite stretch can stretch residual images with negative values:
+
+.. plot::
+    :include-source:
+    :align: center
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+    from astropy.visualization.stretch import AsinhStretch, \
+        LinearStretch, SinhStretch
+    from astropy.visualization import ImageNormalize
+    # For nicer colorbars to show the stretch
+    from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+    # Streches to transform normalized values [0,1] to [-1,1] and back
+    stretch_signed = LinearStretch(a=2, b=-1)
+    stretch_unsigned = LinearStretch(a=0.5, b=0.5)
+
+    # sinh and arcsinh stretches with default values
+    stretches = {
+        'sinh(a=1/3)': stretch_unsigned + SinhStretch(1/3) + stretch_signed,
+        'linear': LinearStretch(),
+        'asinh(a=0.2)': stretch_unsigned + AsinhStretch(0.2) + stretch_signed,
+    }
+
+    # Image of random Gaussian noise
+    img = np.random.normal(size=(64, 64))
+    fig, axes = plt.subplots(1, len(stretches), figsize=(16, 16))
+    for (title, stretch), axis in zip(stretches.items(), axes):
+        norm = ImageNormalize(stretch=stretch)
+        imgax = axis.imshow(img, norm=norm, cmap='gray', vmin=-5, vmax=5)
+        axis.set_title(title)
+        cax = make_axes_locatable(axis).append_axes('right', size='5%', pad=0.1)
+        plt.colorbar(imgax, cax=cax, orientation='vertical')
```

Comment:
This example is a little complicated at the moment - is there any way you could make it simpler by just showing one panel and one composite stretch?